File: app/src/main/java/com/example/amit/uniconnexample/Model/Message_model.java
package com.example.amit.uniconnexample;

/**
 * Created by amit on 23/12/16.
 */

public class Message_model {
    String image,msg,name,key;
    public Message_model(){

    }
    public Message_model(String image, String msg,String name,String key){
        this.image=image;
        this.msg=msg;
        this.name=name;
        this.key=key;
    }

    public String getImage() {
        return image;
    }

    public String getMsg() {
        return msg;
    }

    public String getName() {
        return name;
    }

    public String getKey() {
        return key;
    }
}

File: app/src/main/java/com/example/amit/uniconnexample/Model/Message_model.kt
package com.example.amit.uniconnexample

/**
 * Created by amit on 23/12/16.
 */

class Message_model {
    var image: String? = null
        internal set
    var msg: String? = null
        internal set
    var name: String? = null
        internal set
    var key: String? = null
        internal set

    constructor() {

    }

    constructor(image: String, msg: String, name: String, key: String) {
        this.image = image
        this.msg = msg
        this.name = name
        this.key = key
    }
}
